spec/features/posting_messages_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'posting messages' do
+  scenario 'post a message' do
+    visit('/')
+    click_link('Post a new peep')
+    fill_in('author', with: 'Victor')
+    fill_in('body', with: 'Hello world!')
+    fill_in('date', with: '2020-10-02 10:00:00')
+    click_button('post')
```

Comment:
**Suggestion**
You could simplify steps 4-8 by creating a helper_method.rb file which encapsulates all these steps.

Reply:
Definitely right!